deploy/helm/groundlight-edge-endpoint/templates/tests/test-cloud-connection.yaml

```diff
@@ -0,0 +1,27 @@
+# Test to see if edge-endpoint can proxy to the cloud
```

Comment:
nice

Reply:
Your comment reminded me that I hadn't included these helm tests in the GHA. I've added them now.